### DISCUSSION TITLE
My Github Account Got suspended

### DISCUSSION THREAD

**Discussion by SAHELISAHAA** [community/community#134348] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi everyone,

I hope this message finds you well. I'm reaching out to inform you that my primary GitHub account, which I use for all my professional work, has been unexpectedly suspended. This account is crucial for my job, and this suspension could significantly impact my work and ongoing projects.

Account Name: @MAVRICK-1
Ticket ID: 2913646

I've already submitted a support ticket to GitHub, but I would appreciate any advice or assistance from the community to expedite this process.

Thank you for your understanding and support.

Best regards,
Rishi Mondal

**Comment by edujeetX** [comment]
My account is also got suspended suddenly. It's happening with many of others too. GitHub must do something ASAP

**Comment by queenofcorgis** [comment, marked as answer]
Hi @SAHELISAHAA, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
